### PR TITLE
♻️ feat (remix/next): add drupal namespace directory

### PR DESCRIPTION
**Pull request by jmolivas**
This PR introduce a `drupal` directory to use as namespace to wrap component related to Drupal 

### Components
image

### GraphQL
image

### Components
image

### GraphQL
image

### CODE REVIEW

File: examples/next-graphql/components/helpers/ComponentResolver.tsx [hunk 1]
```diff
@@ -1,12 +1,13 @@
...
 
-import ParagraphHero from "@/components/paragraph/ParagraphHero";
-import ParagraphCardGroup from "@/components/paragraph/ParagraphCardGroup";
-import ParagraphText from "@/components/paragraph/ParagraphText";
-import ParagraphImage from "@/components/paragraph/ParagraphImage";
-import ParagraphCodeBlock from "@/components/paragraph/ParagraphCodeBlock";
-import ParagraphStaticComponent from "@/components/paragraph/ParagraphStaticComponent";
-import ParagraphViewReference from "@/components/paragraph/ParagraphViewReference";
+// Drupal
+import ParagraphHero from "@/components/drupal/paragraph/ParagraphHero";
```
Comment (enZane) [inline examples/next-graphql/components/helpers/ComponentResolver.tsx:4]: maybe we can barrel this to have it like the next export

File: examples/next-graphql/components/ui/Header.tsx [hunk 2]
```diff
@@ -1,6 +1,6 @@
 export default function Header() {
 
-  // @todo: Fetch menusItems from Drupal
+  // @todo: Fetch menuItems from Drupal
```
Comment (enZane) [inline examples/next-graphql/components/ui/Header.tsx:3]: Do we have this effort captured somewhere?

Comment (jmolivas) [reply examples/next-graphql/components/ui/Header.tsx:3]: Nope we need a tkt , could you @enZane ?

File: examples/next-graphql/components/helpers/ComponentResolver.tsx (hunk 1 above)
Comment (jmolivas) [reply examples/next-graphql/components/helpers/ComponentResolver.tsx:4]: agree on this, not sure if we can have a new PR to introduce DX improvements

File: examples/next-graphql/components/ui/Header.tsx (hunk 2 above)
Comment (enZane) [reply examples/next-graphql/components/ui/Header.tsx:3]: Omw

**Review by enZane — APPROVED** [review]
lgtm

**Pull request merged**